txpipe/test/test_cal.py

```diff
@@ -1,5 +1,94 @@
-from ..utils.calibration_tools import MeanShearInBins
+from ..utils.calibration_tools import MeanShearInBins, ParallelCalibratorMetacal
 import numpy as np
+import mockmpi
+
+
+
+def select_all_bool(data):
+    return np.repeat(True, data['mcal_g2'].size)
+
+def select_all_index(data):
+    return np.arange(data['mcal_g2'].size)
+
+def select_all_where(data):
+    return np.where(data['mcal_g2'] != 68754337.86543434)
```

Comment:
What is this number?

Reply:
Just a random number that will never appear!  I'll rewrite to make more sensible.